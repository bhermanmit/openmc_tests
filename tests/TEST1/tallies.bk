<?xml version="1.0"?>
<tallies>

  <mesh id="1">
    <type>rectangular</type>
    <origin>-10 -10 -00</origin>
    <dimension>1 1 5</dimension>
    <width>20 20 20</width>
  </mesh>
    

  <tally id="1">
    <filters>
      <cell>100</cell>
      <mesh>1</mesh>
    </filters>

    <macros>total scatter nu-scatter absorption fission nu-fission</macros>
  </tally>

  <tally id="2">
    <filters>
      <cell>100</cell>
      <energy>0.0 20.0</energy>
    </filters>

    <macros>total scatter nu-scatter absorption fission nu-fission</macros>
  </tally>

  <tally id="3">
    <filters>
      <cell>100</cell>
      <energy>0.0 20.0</energy>
      <energyout>0.0 20.0</energyout>
    </filters>

    <macros>scatter nu-scatter scatter-1 scatter-2 scatter-3</macros>
  </tally>

</tallies>
