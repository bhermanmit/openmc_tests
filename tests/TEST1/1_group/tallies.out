
 ============================>     TALLY 1     <============================

 Cell 1
   Total Reaction Rate         5.28686        +/- 2.49752E-03
   Scattering Rate             4.28322        +/- 2.07380E-03
   Scattering Production Rate  4.28686        +/- 2.49752E-03
   Absorption Rate             1.00364        +/- 4.30837E-04
   Fission Rate                0.928097       +/- 3.81130E-04
   Nu-Fission Rate             2.66739        +/- 1.06396E-03
